### ISSUE TITLE
Add warning re PCI numbering changes to "Devices" tab in QM 

### ISSUE THREAD

**Issue by unman** [QubesOS/qubes-issues#8127]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)
PCI numbering may change if devices are added or altered.(Cf #7792)
This means that device allocation to qubes may change.

### The solution you'd like
Absent a full solution at least provide warning when device allocation is made.
A warning could also be made in the docs.

### The value to a user, and who that user might be
Any user who may make hardware changes will have been warned of possible issues when they first make device allocations.
Whether they remember that warning is a separate issue.

Cf Forum discussion - https://forum.qubes-os.org/t/usability-issues-with-hardware-changes-on-system-qubes/17676

**Comment by marmarta** [comment]
Hm, I get the idea, but also: people will 100% forget this warning. Nobody remembers warnings from - possibly - months before.

**Comment by DemiMarie** [comment]
@marmarek @HW42 Can assignment be based on PCIe path rather than bus-slot-function?

**Comment by deeplow** [comment]
Yes a warning probably not the way to go about this. A technical solution is much better than documentation or notes through the user interface.

And I see that sometimes PCIe devices numbering (within the VM) is randomized at boot of the host. This makes some of my HVMs sometimes boot and other times not boot (the passed-through graphics card is assigned to one slot). 

@neowutran provides a way to circumvent that (changing the Xorg config on boot) [here](https://github.com/Qubes-Community/Contents/blob/master/docs/customization/gaming-hvm.md#linux-guest-----integration-with-qubesos), but that's cumbersome. In my case I just connect the qube to a netVM (from none), which adds a virtual PCIe network card before, making the GPU one be in the right place.

A permanent solution to this (having some sort of PCIe numbering attribution) would make it easier to use. Maybe the slots could be user-assignable? Here's a (terrible) mockup of what I mean:

![penpot](https://github.com/QubesOS/qubes-issues/assets/47065258/b5797d31-3372-4e1f-b818-5c7db192131c)

(for those who can't see the picture, it's basically  a lost of devices on the left and numbered slots on the right, where the user can drag and drop the devices).

**Comment by marmarta** [comment]
PCI devices numbering  no longer changes  (https://github.com/QubesOS/qubes-issues/issues/8681 )
